app/src/main/res/layout/fragment_list_item.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Wrong file naming, this layout is not directly related to fragment, it is the layout for adapter. The convention is to name it `item_*` for ViewHolder layouts.